flocker/node/test/test_deploy.py

```diff
@@ -659,18 +748,8 @@ def test_discover_one(self):
         with a a list of running ``Application``\ s; one for each active
         container.
         """
-        fake_docker = FakeDockerClient(units={APP_NAME: UNIT_FOR_APP})
```

Comment:
:grin: yea I think I might want to do something similar with a few other chunks of code in here... this seemed like a good start.